server/src/main/java/de/uftos/services/StudentGroupService.java

```diff
@@ -215,6 +237,12 @@ public void delete(String id) {
     }
     gradeRepository.saveAll(grades);
 
+    new ConstraintInstanceDeleter(constraintSignatureRepository, constraintInstanceRepository)
```

Comment:
Deletion of relationship with students seems to be missing. (Probably cascading? But seems to be problematic when testing.)

Reply:
I couldn't find any problems with the deletion of student groups. How can I replicate it?

Reply:
Yes, `StudentGroup` is the owner of the relationship with `Student`. So it should automatically cascade on delete